src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/stats/autocomplete.test.ts

```diff
@@ -834,6 +839,64 @@ describe('STATS Autocomplete', () => {
 
           expect(suggestions).toContainEqual(expectedCompletionItem);
         });
+
+        test('should NOT appear inside BUCKET function', async () => {
```

Comment:
It took me some time to understand. This is a suggestion, change it if you dont like it but we need to adjust it

```suggestion
        test('BUCKET constant arguments should not trigger function suggestions', async () => {
```